src/main/java/com/google/sps/servlets/TripServlet.java

```diff
@@ -81,6 +81,7 @@
   private String tripName;
   private String tripDestination;
   private String tripDayOfTravel;
```

Comment:
Change this variable to `tripStartDate` to match the new format of start / end date instead of `tripDayOfTravel`.

Reply:
There are other instances of this -- I won't leave a comment for every instance though.

Reply:
Decided against this as this variable appears in multiple places and in multiple forms (e.g. INPUT_DAY_OF_TRAVEL and tripDayOfTravel)